README.md

```diff
@@ -13,51 +17,59 @@ npm run build
 npm run preview
 ```
 
-## Registration
+## Running complete tool in Docker
 
-Tool also supports configuration to disable registration of new user within security, but by default, everybody can register.
-There are two flags which need to be set:
+1. Change the working directory using `cd ./deploy/no-proxy`
+2. Run `docker-compose up -d` to start the application.
+3. The application should be accessible at `http:/localhost:8080/`
 
-- Backend - Run application with `-Dspring.profiles.active=admin-registration-only` profile
-- Frontend - Set `FTA_FMEA_APP_ADMIN_REGISTRATION_ONLY` in [.env.production](.env.production) to true
 
-## General Info
+## Configuration
 
-We use [Prettier](https://prettier.io/) to keep the codebase formatting consistent. To simplify this process, a pre-commit Git hook is set up with [Husky](https://github.com/typicode/husky).
-This hook runs Prettier code formatting before every commit.
+The tool also supports configuration to disable registration of new users within security, but by default, everybody can register.
+There are two flags which need to be set:
 
-### Pre-commit troubleshooting
+- Backend - Run application with `-Dspring.profiles.active=admin-registration-only` profile
+- Frontend - Set `FTA_FMEA_APP_ADMIN_REGISTRATION_ONLY` in [.env.production](.env.production) to true
 
-If you experience issues on pre-commit using the GUI and that you are using a node version manager, such as nvm, add the following script:
+## Development
 
-```
-# ~/.config/husky/init.sh
-export NVM_DIR="$HOME/.nvm"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
+To develop the tool, the easiest way to run it is as follows:
+1. Run the complete tool as described in [Running complete tool in Docker](#running-complete-tool-in-docker)
```

Comment:
This is confusing to me, did you mean to configure the same way? IN this way, it sounds to me like you have to follow the same steps, which is not the case.

Reply:
Or you meant to do step 1 OR step 2?

Reply:
Maybe what you are asking is described below at the line 45:
![image](https://github.com/kbss-cvut/fta-fmea-ui/assets/3962237/c445a9b8-0792-41e6-8dbd-974c1ecf4bf7)

is it?

Reply:
Yes, I meant this, I did not see it before so I think it's ok to leave it like that